virtool/account/api.py

```diff
@@ -139,11 +123,11 @@ async def get(self) -> Union[r200[AccountSettingsResponse], r401]:
             200: Successful operation
             401: Requires authorization
         """
-        account_settings = await get_one_field(
-            self.request.app["db"].users, "settings", self.request["client"].user_id
+        account_settings = await get_data_from_req(self.request).account.get_settings(
+            "settings", self.request["client"].user_id
         )
 
-        return json_response(account_settings)
+        return json_response(AccountSettingsResponse.parse_obj(account_settings).dict())
```

Comment:
Not necessary to call `dict()`. The `json_response` function will convert for you.
```suggestion
        return json_response(AccountSettingsResponse.parse_obj(account_settings))
```